src/postgres.rs

```diff
@@ -460,7 +460,7 @@ mod diesel_postgres {
 }
 
 #[cfg(any(feature = "db-postgres", feature = "db-tokio-postgres"))]
-mod postgres {
+mod _postgres {
```

Comment:
Clippy stuff. Feel free to choose another name if desired

Reply:
I had this in another project too. for now this should be fine since it's a private mod.